libs/sdk-ui-kit/src/Header/generateHeaderAccountMenuItems.ts

```diff
@@ -0,0 +1,45 @@
+// (C) 2007-2021 GoodData Corporation
+import { IHeaderMenuItem } from "@gooddata/goodstrap/lib/@next";
```

Comment:
this interface should be already in UI SDK, use it

Reply:
thanks I used it instead of using from goodstrap